scripts/travis-extract-l10n.sh

```diff
@@ -0,0 +1,138 @@
+#! /bin/bash
+# This script is supposed to be running automatically via travis cronjobs.
+#
+# This script will do the following:
+#   - prepare git credentials for pull request push
+#   - create a new branch (l10n-extract-2017-08-24-0b3bcaf2ca)
+#   - Update your code
+#   - Extract new strings and push to the .po files
+#
+# This script makes a lot of assumptions and has no error checking, so read it
+# over before you run it.
```

Comment:
redundant warning if "you" don't run it any more?

Reply:
hmm, I'll clarify it a bit. I am updating this PR right now so that the script is perfectly able to run manually and automatically so it should be able to do both.